command/certificate/extractPE.go

```diff
@@ -0,0 +1,168 @@
+package certificate
+
+import (
+	"debug/pe"
+	"encoding/asn1"
+	"encoding/binary"
+	"encoding/pem"
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+	"go.mozilla.org/pkcs7"
+	"io"
+	"os"
+	"path"
+)
+
+var ErrCodeSignCertsNoCertificateTableFound = errors.New("No Certificate Table found")
+
+func extractPECommand() cli.Command {
+	return cli.Command{
+		Name:        "extract-pe",
+		Action:      cli.ActionFunc(extractPEAction),
+		Usage:       `TODO`,
+		UsageText:   `TODO`,
+		Description: `TODO`,
+	}
+}
+
+func extractPEAction(ctx *cli.Context) error {
+	if err := errs.MinMaxNumberOfArguments(ctx, 1, 1); err != nil {
+		return err
+	}
+
+	file, err := os.Open(path.Clean(ctx.Args().Get(0)))
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	f, err := pe.NewFile(file)
+	if err != nil {
+		return errors.Wrap(err, "Unable to open the PE file")
+	}
+	defer f.Close()
+
+	var (
+		buff []byte
+		ob   *asn1.ObjectIdentifier
+	)
+
+	switch header := f.OptionalHeader.(type) {
+	case *pe.OptionalHeader32:
+		buff, ob, err = CodeSignCertsParseHeader32(header, file)
+	case *pe.OptionalHeader64:
+		buff, ob, err = CodeSignCertsParseHeader64(header, file)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	err = CodeSignCertsParseCertificateTable(buff, ob, os.Stdout)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Wrapper Functions
+// PE headers can be written by 64 bits architectures or 32 bits architectures.
+// 32 bits is IMHO more spread out because it's backward compatible with 64 bits.
+// I don't think it changes the fact that the program can for a 64 bits arch and the headers 32 bits.
+func CodeSignCertsParseHeader64(header *pe.OptionalHeader64, file io.ReaderAt) ([]byte, *asn1.ObjectIdentifier, error) {
```

Comment:
Use the standard way to comment public methods (or make them private)
```
FuctionName blah blah blah
```

Add comment to all public methods, if not the linter will complain.